test/katello_test_helper.rb

```diff
@@ -1,17 +1,24 @@
 require 'simplecov'
 require 'simplecov-rcov'
-SimpleCov.formatters = [
-  SimpleCov::Formatter::RcovFormatter,
-  SimpleCov::Formatter::HTMLFormatter
-]
-SimpleCov.start 'rails'
 
 require 'test_helper'
 require 'factory_girl_rails'
 require "webmock/minitest"
 require "mocha/setup"
 require 'set'
 
+SimpleCov.formatters = [
+  SimpleCov::Formatter::RcovFormatter,
+  SimpleCov::Formatter::HTMLFormatter
+]
+
+SimpleCov.start do
+  filters.clear
+  add_filter do |src|
+    !(src.filename =~ /^#{SimpleCov.root}/) unless src.filename =~ /katello/
```

Comment:
I think if you change this line to just `src.filename =~ /katello/`, it should exclude the foreman files.